pkg/config/middleware.go

```diff
@@ -120,6 +128,162 @@ func setupRecoveryMiddleware() *negroni.Recovery {
 	return r
 }
 
+type OidcConfiguation struct {
+	JwkURI string `json:"jwks_uri"` // This is the only field we care about for now.
+}
+
+type OidcJwk struct {
```

Comment:
Can we add a golang style comment like "OidcJwk is a ..."? Same applies to all public exposed symbols.